CMakeLists.txt

```diff
@@ -432,8 +432,8 @@ OCV_OPTION(WITH_IMGCODEC_PXM "Include PNM (PBM,PGM,PPM) and PAM formats support"
 OCV_OPTION(WITH_QUIRC "Include library QR-code decoding" ON
   VISIBLE_IF TRUE
   VERIFY HAVE_QUIRC)
-OCV_OPTION(WITH_TENGINE "Include Arm Inference Tengine support" OFF
-  VISIBLE_IF (ARM OR AARCH64) AND UNIX AND NOT ANDROID AND NOT IOS
+OCV_OPTION(WITH_TENGINE "Include Arm Inference Tengine support" ON
```

Comment:
@liqi-c @vpisarev 
Is it a testing mode for PR?

Reply:
I'm sorry , i close the test .